1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -83,17 +85,30 @@ function generate_kubernetes_config() {
     else
       return
     fi
-    [ -z "${slave_templates}" ] && return
     echo "
-    <org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud plugin=\"kubernetes@0.5\">
+    <org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud plugin=\"kubernetes@0.6\">
```

Comment:
yeah i'm bothered we need this at all. i'm going to check if it's really required because it's annoying.